org.eclipse.buildship.core/src/main/resources/org/eclipse/buildship/core/i18n/CoreMessages.properties

```diff
@@ -49,4 +49,5 @@ ErrorMessage_0_MustBeDirectory={0} must be a directory.
 ErrorMessage_0_AlreadyExists={0} already exists.
 
 Preference_Label_GradleUserHome=Gradle User Home
-Preference_Label_OfflineMode=Offline mode
\ No newline at end of file
+Preference_Label_OfflineMode=Offline mode
+Preference_Label_BuildScans=Enable Build Scans
```

Comment:
This might need some more explanation. Can we maybe add some "info" bubble next to it where we explain what exactly this option does?